apps/site/src/app/(main)/(home)/sections/ChooseCharacter/CharacterBox.tsx

```diff
@@ -0,0 +1,65 @@
+import type { ComponentProps } from "react";
+import Image, { StaticImageData } from "next/image";
+import Link from "next/link";
+
+import centerChatBox from "@/assets/images/center_chat_box.svg";
+import centerChatBG from "@/assets/images/center_chat_box_bg.svg";
+
+import styles from "./CharacterBox.module.scss";
+
+interface CharacterBoxProps {
+	className?: string;
+	imageSrc: ComponentProps<typeof Image>["src"];
+	chatText: string;
+	titleText: string;
+	clipClass: string;
+	chatBoxImageSrc: StaticImageData;
+	bgImageSrc: StaticImageData;
+	textAlign: string;
+	href: string;
+}
+
+export default function CharacterBox({
+	className,
+	imageSrc,
+	chatText,
+	titleText,
+	clipClass,
+	chatBoxImageSrc,
+	bgImageSrc,
+	textAlign,
+	href,
+}: CharacterBoxProps) {
+	return (
+		<div
+			className={`${className} ${styles.hoverGroup} flex flex-col items-center justify-center mb-20 lg:mb-0`}
+		>
+			<div className="relative">
+				<Image className="absolute hidden lg:block" src={bgImageSrc} alt="" />
+				<Image className="absolute lg:hidden" src={centerChatBG} alt="" />
```

Comment:
Suggestion [repeating]: give the sprites and images of the text boxes some alt text